blead/patches/003.patch

```diff
@@ -84,7 +84,16 @@ index 8995d8c..6e7098d 100644
  
  def_os390_defs="$def_os390_defs -DMAXSIG=39 -DNSIG=39";     # maximum signal number; not furnished by IBM
  def_os390_defs="$def_os390_defs -DOEMVS";   # is used in place of #ifdef __MVS__
-@@ -234,10 +248,10 @@ esac
+@@ -96,6 +110,8 @@ def_os390_defs="$def_os390_defs -DNO_LOCALE_MESSAGES"
+ # Set up feature test macros required for features available on supported z/OS systems
+ def_os390_defs="$def_os390_defs -D_OPEN_THREADS=3 -D_UNIX03_SOURCE=1 -D_AE_BIMODAL=1 -D_XOPEN_SOURCE_EXTENDED -D_ALL_SOURCE -D_ENHANCED_ASCII_EXT=0xFFFFFFFF -D_OPEN_SYS_FILE_EXT=1 -D_OPEN_SYS_SOCK_IPV6 -D_XOPEN_SOURCE=600 -D_XOPEN_SOURCE_EXTENDED"
+ 
++startperl='#!/bin/env perl'
```

Comment:
can you add a comment explaining why this is necessary for relocation?

Reply:
Added comment above